src/metabase/driver/sql_jdbc/sync/describe_database.clj

```diff
@@ -92,27 +109,33 @@
          (.rollback conn))
        false))))
 
+(defn- jdbc-get-tables
```

Comment:
I'm not sure why this function needs to exist separately from db-tables anymore. I think it's a relic of the past version where you wanted its signature to match `get-tables`?

Reply:
hmm I thought I had a note for it some where, but I left it as its own function for testing purposes, see the test I have in `test/metabase/driver/postgres_test.clj`

Reply:
Yeah what I'm not understanding is why test `jdbc-get-tables` when you can test `db-tables`? I thought you did this because originally `get-tables` had a `types` argument, but now it doesn't.

What we actually want to test is that `postgres/get-tables` returns the same thing as `db-tables`, because `fast-active-tables` uses `db-tables`. `jdbc-get-tables` feels more like an implementation detail. The fact that you've pasted the exact list of types that postgres needs to return [here](https://github.com/metabase/metabase/pull/39670/commits/b5b9b8a6f7cb305afe85b25e251889d441a6a838#diff-821f0be02cebbea5f2f5b86239bd9df5e5cb4f671634a761fa6da1c511487125R1378) effectively makes that test equivalent to testing `db-tables` anyway. But it would be clearer if you just used `db-tables` instead.

Reply:
cool, that makes sense